sdk/python/symbolchain/symbol/ExtendedTypeParsingRules.py

```diff
@@ -6,8 +6,7 @@
 def extend_type_parsing_rules(symbol_type_converter, base_type_parsing_rules=None):
 	builder = TypeParserBuilder('sc', symbol_type_converter, base_type_parsing_rules)
 
-	# note, some of those below could be automatically generated,
-	# but right now symbol.yml contains also state-related objects
+	# note, some of those below could be automatically generated
```

Comment:
@gimre-xymcity - not sure what this comment should say as there is no symbol.y(a)ml

Reply:
it's fine, we could generate (some/ most) of the rules based on AST iterate through all things, 
 - for pods, flags and enums we could probably always fire rule
 - for structs - only those that are in hints of transactions